### PR TITLE
NF-48 - Address page

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala [hunk 1]
```diff
@@ -16,18 +16,17 @@
...
-
 object Validation {
 
-  val accountNumberPattern: Regex = "^[ -]*(?:\\d[ -]*){6,8}$".r.anchored
-  val sortCodePattern: Regex      = "^[ -]*(?:\\d[ -]*){6,6}$".r.anchored
-  val safeInputPattern            = """^[A-Za-z0-9À-ÿ \!\)\(.,_/’'"&-]+$"""
-  val entryProcessingUnit         = "^[0-9][0-9][0-9]$"
-  val entryNumber                 = "^([0-9]{6}[a-z|A-Z])$"
+  val accountNumberPattern = "^[0-9]{6,8}$"
+  val sortCodePattern      = "^[0-9]{6}$"
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:22]: Restricting account number and sort code to match schema until we decide common approach for validating "complex" fields

Comment (crosslandwa) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:22]: OBSERVATION: Noted that this change is unrelated to the page this PR pertains to

Happy to approve this PR to include this change (I assume the acceptance tests still pass with this in place) - i'd prefer unrelated changes to go into their own PRs (for traceability)

Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:22]: This is a ninja bug fix.

Comment (SteveSugden) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:22]: ... but happy to move to own PR.  Just getting a ticket ready to explain problem

Comment (crosslandwa) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/forms/mappings/Validation.scala:22]: Its fine on this PR (lets avoid re-work), though if you could link this PR to the ticket (once created) that would be grand

**Review by crosslandwa — APPROVED** [review]

**Pull request merged**